file-formats/edck/type/zif_aff_edck_v1.intf.abap

```diff
@@ -0,0 +1,74 @@
+INTERFACE zif_aff_edck_v1
+  PUBLIC.
+
+  TYPES ty_check_classification TYPE c LENGTH 1.
+  TYPES ty_check_category TYPE c LENGTH 6.
+
+  TYPES:
+    BEGIN OF ty_attributes,
+      "! <p class="shorttext">Check Category</p>
+      "! Consistency check category
+      "! $values {@link zif_aff_edck_v1.data:co_check_category}
+      "! $default {@link if_aff_edck_v1.data:co_check_category.exist}
+      "! $required
+      check_category              TYPE ty_check_category,
+      "! <p class="shorttext">Check Classification</p>
+      "! Consistency check ID classification
+      "! $values {@link zif_aff_edck_v1.data:co_check_classification}
+      "! $default {@link if_aff_edck_v1.data:co_check_classification.core}
+      "! $required
+      check_id_classification     TYPE ty_check_classification,
+      "! <p class="shorttext">Additional Check Class</p>
+      "! Consistency check class
+      check_class                 TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Result Derivation Indicator</p>
+      "! Result process derived by BAdI
+      result_derivation_indicator TYPE abap_bool,
+    END OF ty_attributes.
+
+  TYPES:
+    "! <p class="shorttext">Consistency Check ID</p>
+    "! Consistency check ID (EDCK) v1
+    BEGIN OF ty_main,
+      "! $required
+      format_version TYPE zif_aff_types_v1=>ty_format_version,
+      "! <p class="shorttext">Header</p>
+      "! Header
+      "! $required
+      header         TYPE zif_aff_types_v1=>ty_header_60_cloud,
+      "! <p class="shorttext">Consistency Check ID Definition</p>
+      "! Attributes of a consistency check ID
+      "! $required
+      attributes     TYPE ty_attributes,
```

Comment:
```suggestion
      "! <p class="shorttext">General Information</p>
      "! General information
      "! $required      
      general_information     TYPE ty_general_information,
```
For consistency reasons with other AFFs, we usually prefer the name `general_information` if you don't need a specific name as described in our [wiki](https://github.com/SAP/abap-file-formats/blob/be3f5e582f75b53cc85fcf4ac8a7a212a9be87ec/docs/json.md#writing-json-schema-with-abap-types).
If you are able to change it, please also remember to change the title and description.